087-external-certificate-manager.md

```diff
@@ -0,0 +1,241 @@
+# External Certificate Manager
+
+This proposal aims to allow Strimzi users to use an external certificate manager, specifically [cert-manager](https://cert-manager.io/), to manage certificates.
+
+## Current situation
+
+There are two different categories of certificates that Strimzi handles:
+* The term "cluster" refers to certificates that are issued for the Strimzi components:
+  * ZooKeeper nodes
+  * Kafka nodes
+  * Cluster, User and Topic operators
+  * Cruise Control
+  * Kafka Exporter
+* The term "clients" refers to certificates that are issued for user applications using the User Operator, or through another external mechanism chosen by the user.
+
+For both categories, to provide a secure, TLS-enabled setup by default when deploying Kafka clusters, Strimzi integrated its own CA operations into the Cluster Operator.
+The Cluster Operator accomplishes this by using openssl to generate self-signed root CA certificates and private keys which it then uses to directly sign end-entity (EE) certificates.
+This CA certificate has zero pathlen, which means it cannot sign any intermediate CA.
+A cluster CA and clients CA are generated.
+These CAs are only used for Kafka clusters and a unique instance of each CA is used for each Kafka cluster.
+
+In addition to Strimzi fully managing the certificates as described above, there are options for users to partially manage the certificates:
+* Users can [install and use their own CA certificate and private keys](https://strimzi.io/docs/operators/latest/deploying#installing-your-own-ca-certificates-str), instead of using the defaults generated by the Cluster Operator.
+  When using this option, both the CA certificate and private key must be provided, and Strimzi still issues the end-entity (EE) certificates that are presented by the components.
```

Comment:
Is it worth mentioning here that for the Clients CA, users can also:
* provide the CA public key
* provide a dummy value for the CA private key
* Issue their own user certificates out-of-band
* Use the `type: tls-external` users when they decide to use the KafkaUser resources and User Operator to manage ACLs and Qoutas

I think this pattern is used enough to make it worth mentioning here for clarity.